public/app/features/alerting/unified/components/rule-editor/RuleFolderPicker.tsx

```diff
@@ -59,6 +59,7 @@ export function RuleFolderPicker(props: RuleFolderPickerProps) {
       permissionLevel={PermissionLevelString.View}
       customAdd={customAdd}
       folderWarning={folderWarning}
+      sliceResults={MAX_FOLDERS_IN_LIST}
```

Comment:
Don't need this extra constant if it is set to the default value anyway, wdyt?

Reply:
yes!! I forgot to remove it, thank you! @JoaoSilvaGrafana